### ISSUE TITLE
pacman -S acpi_call     fails to run depmod  because ARCH instead of MANJARO

### ISSUE THREAD

**Issue by ghost** [manjaro/packages-community#1]
TL;DR: and also puts the module in the wrong directory (`/usr/lib/modules/extramodules-3.15-ARCH/`)

Hi.
`cat: /usr/lib/modules/extramodules-3.15-ARCH/version: No such file or directory`
source code causing the error is here: https://projects.archlinux.org/svntogit/community.git/tree/trunk/acpi_call.install?h=packages/acpi_call

the problem is `ARCH` is there in the code instead of `MANJARO`
and this happens in PKGBUILD file also: https://projects.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/acpi_call

which puts the module in this folder:

```bash
$ ls -la /usr/lib/modules/extramodules-3.15-ARCH/
total 12
drwxr-xr-x 2 root root 4096 11.07.2014 02:20 ./
drwxr-xr-x 9 root root 4096 11.07.2014 02:16 ../
-rw-r--r-- 1 root root 3992 01.07.2014 08:37 acpi_call.ko.gz
```
correct folder, in my case, is:

```bash
$ ls -la /usr/lib/modules/extramodules-3.15-MANJARO/
total 3192
drwxr-xr-x 2 root root    4096 11.07.2014 02:01 ./
drwxr-xr-x 9 root root    4096 11.07.2014 02:16 ../
-rw-r--r-- 1 root root 3122433 09.07.2014 04:49 fglrx.ko.gz
-rw-r--r-- 1 root root  128841 01.07.2014 23:02 ndiswrapper.ko.gz
-rw-r--r-- 1 root root      17 09.07.2014 04:33 version

```

I didn't try to make a solution yet because you guys know better how to do that.
Thank you for your time and consideration ;)

**Comment by philmmanjaro** [comment]
We don't support **linux315** anymore. Please update to **linux316** or any other supported kernel. There we have for each kernel [acpi_call](https://github.com/manjaro/packages-extra/tree/master/linux316-extramodules/acpi_call) already compiled.
